assets/timber.scss.liquid

```diff
@@ -1274,33 +1252,12 @@ legend {
   padding: 0;
 }
 
-optgroup {
-  font-weight: bold;
-}
-
-input {
-  display: inline-block;
-  width: auto;
-}
-
 button,
 input[type="submit"] {
   cursor: pointer;
 }
 
-input[type="submit"] {
-  @extend .btn;
-}
-
-/*================ Input width and border ================*/
-input[type="text"],
-input[type="search"],
-input[type="password"],
-input[type="email"],
-input[type="file"],
-input[type="number"],
-input[type="tel"],
-input[type="url"],
+input,
```

Comment:
i don't think we should be this opinionated about form input styling by default. There was a good discussion about this in FED slack earlier today, but by styling form elements directly, if you want to create any variation within the theme for those elements, you'd need to undo all of these styles in that variant.